components/molecules/MarkdownWrapper/markdown-wrapper.tsx

```diff
@@ -0,0 +1,39 @@
+import ReactMarkdown from "react-markdown";
+import SyntaxHighlighter from "react-syntax-highlighter";
+import { atomOneDark } from "react-syntax-highlighter/dist/cjs/styles/hljs";
+
+interface MarkdownWrapperProps {
+  content: string;
+  pClassName?: string;
```

Comment:
```suggestion
  paragraphClassName?: string;
```

Reply:
The suggested styling changes have been committed, thank you for reviewing!